tools/build_board_info.py

```diff
@@ -34,6 +38,8 @@
 extension_by_board = {
     # samd
     "arduino_mkr1300": BIN,
+    "arduino_mkrzero": BIN,
+    "arduino_nano_33_iot": BIN,
```

Comment:
Should these actually be `BIN_UF2`, since a UF2 bootloader exists for the boards?

Reply:
I was going by `arduino_zero`, but we have a UF2 bootloader for that too, so I could make all three be BIN_UF2.